server/index.js

```diff
@@ -18,12 +18,52 @@ const db = require('../database/pg.js');
 
 app.get('/support/:id', (req, res) => {
   let albumId = Number(req.params.id);
-  console.log(typeof albumId);
-  db.getUsersForAlbum(albumId, (err, albumUsers) => {
+
+  db.getAlbumComments(albumId, (err, albumUsers) => {
     if (err) {
       console.log(err);
     } else {
-      res.json(albumUsers);
+      res.send(albumUsers);
+    }
+  });
+});
+
+app.post('/support/:id', (req, res) => {
+  let albumId = Number(req.params.id);
+  let params = [albumId, req.body.username, req.body.comment, req.body.profileImg]
+
```

Comment:
So I was reading up on best practices of designing API routes. It said on creation your route would ideally be `/support` without the id as the id would be created when you insert it into the database. 
https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9 --- This is the resource I read.